applications_tests/gls_navier_stokes_2d/gls_tracer_simplex_diffusion_mms_2d.prm

```diff
@@ -43,7 +43,9 @@ end
 # Physical Properties
 #---------------------------------------------------
 subsection physical properties
-  set tracer diffusivity = 1
+  subsection fluid 0
```

Comment:
I know that the default value of the variable "number of fluids" is equal to 1, but I think it would be better to define this variable in 2-3 application tests as an example.

Reply:
Very good suggestion. I will add it to the 2D tests just to be sure that it is clear.
I'll also modify the wiki documentaiton.